utils/src/field_helpers.rs

```diff
@@ -106,23 +100,25 @@ impl<F: Field> FieldHelpers<F> for F {
             bits
         })
     }
+}
 
-    fn from_big(big: BigUint) -> Result<F> {
-        let mut bytes = big.to_bytes_le();
+/// Field element helpers for [BigUint]
+pub trait BigFieldHelpers<F> {
+    /// Deserialize from big unsigned integer
+    fn from_big(big: BigUint) -> Result<F>;
 
-        // Pad with zeros until multiple of 32 if necessary because the BigUint to_bytes_le function gives the smallest possible vector of bytes
-        if bytes.len() % 32 != 0 {
-            // smallest larger multiple of 32 bytes
-            let goal = 32 + bytes.len() / 32;
-            bytes.extend(std::iter::repeat(0).take(goal - bytes.len()));
-        }
+    /// Serialize field element as a big unsigned integer
+    fn to_big(self) -> BigUint;
+}
 
-        F::deserialize(&mut &bytes[..]).map_err(|_| FieldHelpersError::DeserializeBytes)
+impl<F: PrimeField> BigFieldHelpers<F> for F {
```

Comment:
I had to move them to a new trait because the other one is just for Field. But I would prefer it to be in the same trait as before if I could cast F as PrimeField just for these two.